src/main/java/uk/gov/hmcts/darts/cases/controller/CaseController.java

```diff
@@ -134,17 +132,6 @@ public ResponseEntity<SingleCase> casesCaseIdGet(Integer caseId) {
 
     }
 
-
-    @Override
-    @SecurityRequirement(name = SECURITY_SCHEMES_BEARER_AUTH)
-    @Authorisation(contextId = CASE_ID,
-        securityRoles = {JUDGE, REQUESTER, APPROVER, TRANSCRIBER},
-        globalAccessSecurityRoles = {JUDGE})
-    public ResponseEntity<SingleCase> casesCaseIdPatch(Integer caseId, PatchRequestObject patchRequestObject) {
-        PatchCaseRequestValidator.validate(patchRequestObject);
```

Comment:
does hte PatchCaseRequestValidator not need deleted also?

Reply:
Yeah. Can be removed. Will address.